playbooks/hokusai.md

```diff
@@ -3,202 +3,119 @@ title: Hokusai
-
-_Make sure IAM credentials are set in your shell via AWS_ACCESS_KEY_ID and AWS_SECRET_ACCESS_KEY environment variables or ~/.aws/credentials!_
-
-If you have the [aws cli](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-welcome.html) installed, you can set up your ` ~/.aws/config` and ` ~/.aws/credentials` files with the command `aws configure`
-
-Also make sure you set your default region to `us-east-1` either via config or by setting the environment variable `AWS_DEFAULT_REGION=us-east-1`
-
 ```
-hokusai configure --kubectl-version 1.19.16 --s3-bucket artsy-citadel --s3-key k8s/config-dev
+HOKUSAI_GLOBAL_CONFIG=s3://artsy-provisioning-public/hokusai/hokusai-dev.yml hokusai configure
 ```
 
-#### Note
+It will install `kubectl` to `~/.local/bin` which can be overridden by command line options. Whichever directory you choose, please ensure it is in your PATH. We advise to avoid using `/usr/local/bin`, as [Docker Desktop uses that for its own `kubectl`](https://github.com/artsy/hokusai/issues/349) which doesn't work with our Kubernetes clusters.
 
-- Due to [recent PyPi upgrades](https://gis.stackexchange.com/questions/278989/pip-no-longer-works-with-qgis-2-18-x-python-tls-version-no-longer-supported)
-`pip install --upgrade hokusai` may complain about SSL cipers being out of date. To fix this, make sure your openssl libraries are up-to-date: `brew upgrade openssl` and reinstall python via `brew reinstall python`.
+Please see [hokusai configure's command reference](https://github.com/artsy/hokusai/blob/main/docs/Command_Reference.md#configuring-hokusai-for-your-organization) to learn more.
 
-- The artsy-citadel S3 bucket isn't open source, and the above will fail for non-Artsy team-members.
 
-### Setting up a new project
+## Using Hokusai to set up a new Artsy project
 
-We keep templates for bootstrapping new projects in https://github.com/artsy/artsy-hokusai-templates
+We have [templates](https://github.com/artsy/artsy-hokusai-templates) for bootstrapping new Artsy projects. The templates currently cover Rails and NodeJS projects.
 
-Set up a new Rails/Puma project with:
+To set up a Rails/Puma project:
 
 ```
 cd ./path/to/my/rails/project/git/repo
 hokusai setup --template-remote git@github.com:artsy/artsy-hokusai-templates.git --template-dir rails-puma
 ```
 
-Set up a new NodeJS project with:
+To set up a NodeJS project:
 
 ```
 cd ./path/to/my/node/project/git/repo
 hokusai setup --template-remote git@github.com:artsy/artsy-hokusai-templates.git --template-dir nodejs
 ```
 
-### Developing with Hokusai
-
-Use `hokusai dev` sub-commands to work with the Docker-Compose development environment defined in
-`./hokusai/development.yml`. Modify this file to suit your project's specific needs and dependencies.
-
-### Testing with Hokusai
-
-Use `hokusai test` to run your test suites in the Docker-Compose test environment defined in `./hokusai/test.yml`.
-Modify this file to suit your project's specific needs and dependencies.
```

Comment:
these sections on dev/test add no value, same info as in [hokusai's command ref](https://github.com/artsy/hokusai/blob/main/docs/Command_Reference.md).